Configuration/Geometry/python/GeometryExtended2023D17Reco_cff.py

```diff
@@ -19,11 +19,11 @@
 from Geometry.CaloEventSetup.CaloTopology_cfi import *
 from Geometry.CaloEventSetup.CaloGeometryBuilder_cfi import *
 CaloGeometryBuilder = cms.ESProducer("CaloGeometryBuilder",
-    SelectedCalos = cms.vstring("HCAL"                   ,
-                                "ZDC"                    ,
-                                "EcalBarrel"             ,
-                                "TOWER"                  ,
-                                "HGCalEESensitive"       ,
+    SelectedCalos = cms.vstring("HCAL",
```

Comment:
hi @kpedro88 - just to be sure I understand - this is an example of the change triggering the need for the change you suggest?

Reply:
Yes - the unit test is intended to prevent the generated .py files from being edited manually. The dictionary contents have to be in sync with the generated .py files for the Phase2 geometries.